src/start/AppStoreLink.android.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
Hmm.. it actually looks like the only difference between `AppStoreLink.android.js` and `AppStoreLink.ios.js` is the image source, so maybe it's better if we merge these into one file (`AppStoreLink.js`) and choose the image source based on the platform.

I like having this as a separate component though!